package.json

```diff
@@ -31,7 +31,7 @@
     "tmp-sync": "^1.0.0"
   },
   "devDependencies": {
-    "ember-cli": "~2.16.0",
+    "ember-cli": "github:ember-cli/ember-cli",
```

Comment:
Can this reference a specific version, or does it still need to be pinned to canary?

Reply:
It still needs to be pinned to canary for now, specifically this ember-cli's PR https://github.com/ember-cli/ember-cli/pull/7961 (not yet on beta branch).

Reply:
Chatted with @rwjblue and he is 👍 on this, at least as a temporary solution.

Reply:
Great! Thank you @mixonic. I'll update to a published ember-cli version when available.

Reply:
Do you think it could be released?